### PR TITLE
Added returnIterable method.

### PR DESCRIPTION
**Pull request by inlimbo**
returns copy of object as iterable. consumable by for..of loops :dancer:

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -1,78 +1,84 @@
...
 
-  function forceKill(prop) {
+  function forceKill (prop) {
     kill(prop)
     return force()
   }
 
+  function returnIterable () {
+    return (function * (obj) {
+      for (let key of Object.keys(obj)) yield ({key: key, value: obj[key]})
```
Comment (hzlmn) [inline lib/index.js]: Why not use cached `Object.keys` here ?

**Comment by hzlmn** [comment]
Thanks for PR, good idea, but including generators transform adds extra dependency `babel-runtime` and lib becomes much bigger. These kind of behavior could be provided by simple `iterator` pattern.

File: lib/index.js (hunk 1 above)
Comment (inlimbo) [inline lib/index.js]: true that, but generator felt like obvious way to go. maybe I should reimplement it with a simple iterator solution?

Comment (inlimbo) [inline lib/index.js]: oops. sorry

**Comment by inlimbo** [comment]
generator implementation a straight forward solution. Maybe, i should reimplement it without generators? :stuck_out_tongue:

**Comment by hzlmn** [comment]
@inlimbo Yes, please. It would be good.

**Pull request closed without merging**